test/mango.py

```diff
@@ -32,8 +33,7 @@ def __init__(self, host, port, dbname, auth=None):
         self.port = port
         self.dbname = dbname
         self.sess = requests.session()
-        if auth is not None:
-            self.sess.auth = auth
+        self.sess.auth = ('adm', 'pass')
```

Comment:
I added it to the readme, and re-instated the skip tests along with the whitespace issues.